invenio_rdm_records/serializers.py

```diff
@@ -23,6 +23,19 @@ def _serialize_access_right(self, obj):
                 category=obj["access"]["access_right"]
             ))
 
+    def _serialize_aggregations(self, obj_list):
+        """Dump ui config for aggregations."""
+        aggregations = obj_list.get("aggregations")
+        if aggregations:
+            aggregations["ui"] = dict(
+                access_right=dict(
+                    open="Open Access",
```

Comment:
Text needs to be marked for translation. ``_(....)``

Shelved: we likely need this as a vocabulary.....we need to think a bit more about it, so just merge for now